aspnetcore/fundamentals/openapi/includes/openapi9.md

```diff
@@ -0,0 +1,56 @@
+:::moniker range=">= aspnetcore-9.0"
+ASP.NET Core supports the generation of OpenAPI documents in controller-based apps and minimal APIs.
+The [OpenAPI specification](https://spec.openapis.org/oas/latest.html) is a programming language-agnostic standard for documenting HTTP APIs. This standard is supported in ASP.NET Core apps through a combination of built-in APIs and open-source libraries. There are three key aspects to OpenAPI integration in an application:
+
+* Generating information about the endpoints in the app.
+* Gathering the information into a format that matches the OpenAPI schema.
+* Exposing the generated OpenAPI schema via a visual UI or a serialized file.
+
+ASP.NET Core apps provide built-in support for generating information about endpoints in an app via the `Microsoft.AspNetCore.OpenApi` package.
+
+The following code is generated by the ASP.NET Core minimal web API template and uses OpenAPI:
+
+[!code-csharp[](~/fundamentals/minimal-apis/9.0-samples/WebMinOpenApi/Program.cs?name=snippet_default&highlight=5,9-12)]
+
+In the preceding highlighted code:
+
+* `AddOpenApi` registers services required for OpenAPI document generation into the application's DI container.
+* `MapOpenApi` adds an endpoint into the application for viewing the OpenAPI document serialized into JSON.
```

Comment:
```suggestion
* `MapOpenApi` adds an endpoint into the application for viewing the OpenAPI document serialized into JSON. The OpenAPI endpoint is restricted to the development environment to minimize the risk of exposing sensitive information and reduce the attack surface in production.
```

Reply:
I think we should explicitly point out why this is for the development environment only whenever we can.

Reply:
AFAIK that applies only to SwaggerUI, not to MapOpenApi